index.html

```diff
@@ -4546,44 +4545,58 @@ <h2 id="AudioWorklet-concepts">
               <a><code>AudioWorkletProcessor</code></a>
             </figcaption>
           </figure>
-          <blockquote>
-            <pre>
-// In the main global scope
-window.audioWorklet.import("bypass.js").then(function () {
-  var context = new AudioContext();
-  var bypass = new AudioWorkletNode(context, "Bypass");
-});
-</pre>
-          </blockquote>
           <p>
-            Importing a JavaScript file via <code>import(url)</code> method
-            registers class definitions of
-            <a><code>AudioWorkletProcessor</code></a> under the
-            <a><code>AudioWorkletGlobalScope</code></a>. Imported definitions
-            are stored in the <a><em>node name to processor definition
-            map</em></a> and any <a><code>AudioContext</code></a> instance from
-            the associated main global scope can access this map to create a
-            custom node.
+            Importing a JavaScript file via <a href=
+            "https://drafts.css-houdini.org/worklets/#dom-worklet-import">import(moduleUrl)</a>
+            method registers class definitions of <a>AudioWorkletProcessor</a>
+            under the <a>AudioWorkletGlobalScope</a>. There are two internal
+            storage areas for the imported class definitions and the active
+            instances created from the definition.
           </p>
-          <blockquote>
-            <pre>
-// 'bypass.js' in AudioWorkletGlobalScope
+          <ul>
+            <li>
+              <dfn>node name to processor definition map</dfn><br>
+              Belongs to <a>AudioWorkletGlobalScope</a>. This map stores
+              definitions of <a>AudioWorkletProcessor</a> with the associated
+              string key. Initially it is empty. It becomes populated when
+              <a href=
+              "widl-AudioWorkletGlobalScope-registerProcessor-void-DOMString-name-VoidFunction-processorConstructor">
+              registerProcessor</a> method is called.
+            </li>
+            <li>
+              <dfn>node instance set</dfn><br>
+              Belongs to an instance of <a>BaseAudioContext</a>. This set
+              stores the reference to all the active instances of
+              <a>AudioWorkletNode</a> that are created by the associated
+              <a>BaseAudioContext</a>. The context keeps the track of instances
+              until they are destroyed by UA.
+            </li>
+          </ul>
+          <pre class="example" title=
```

Comment:
This is to provide the conceptual overview of AudioWorklet system without going into the detail. FWIW, this change was made based on @domenic's feedback.